piet-coregraphics/src/ct_helpers.rs

```diff
@@ -388,6 +388,12 @@ impl FontCollection {
 extern "C" {
     static kCTFontFamilyNameKey: CFStringRef;
 
+    pub static kCTFontVariationAxisIdentifierKey: CFStringRef;
+    //static kCTFontVariationAxisMinimumValueKey: CFStringRef;
```

Comment:
Generally commented out code is something we would want to avoid, but in this particular case I see the value - these are things we may want to add later.